src/Batteries.purs

```diff
@@ -103,117 +147,1161 @@ module Batteries
+  , cardinality
+  , defaultFromEnum
+  , defaultPred
+  , defaultSucc
+  , defaultToEnum
+  , enumFromThenTo
+  , enumFromTo
+  , fromEnum
+  , intFromTo
+  , intStepFromTo
+  , pred
+  , runCardinality
+  , succ
+  , toEnum
+  )
 import Data.Equivalence
+  (
+  )
 import Data.Exists
+  (
+  )
 import Data.Foldable
+  ( Foldable
+  , all
+  , and
+  , any
+  , elem
+  , find
+  , fold
+  , foldl
+  , foldMap
+  , foldr
+  , for_
+  , intercalate
+  , mconcat
+  , notElem
+  , or
+  , product
+  , sequence_
+  , sum
+  , traverse_
+  )
 import Data.Foreign
+  (
+  )
 import Data.Foreign.Class
+  (
+  )
+import Data.Foreign.Index
+  (
+  )
+import Data.Foreign.Keys
+  (
+  )
+import Data.Foreign.Null
+  (
+  )
+import Data.Foreign.NullOrUndefined
+  (
+  )
+import Data.Foreign.Undefined
+  (
+  )
 import Data.Function
+  ( Fn0 ()
+  , Fn1 ()
+  , Fn10 ()
+  , Fn2 ()
+  , Fn3 ()
+  , Fn4 ()
+  , Fn5 ()
+  , Fn6 ()
+  , Fn7 ()
+  , Fn8 ()
+  , Fn9 ()
+  , mkFn0
+  , mkFn1
+  , mkFn10
+  , mkFn2
+  , mkFn3
+  , mkFn4
+  , mkFn5
+  , mkFn6
+  , mkFn7
+  , mkFn8
+  , mkFn9
+  , on
+  , runFn0
+  , runFn1
+  , runFn10
+  , runFn2
+  , runFn3
+  , runFn4
+  , runFn5
+  , runFn6
+  , runFn7
+  , runFn8
+  , runFn9
+  )
 import Data.Functor
+  ( (<$)
+  , ($>)
+  )
 import Data.Functor.Contravariant
+  (
+  )
 import Data.Functor.Contravariant.Divisible
+  (
+  )
 import Data.Functor.Coproduct
+  (
+  )
 import Data.Functor.Invariant
+  ( Invariant
+  , imap
+  , imapF
+  )
 import Data.Generic
+  (
+  )
 import Data.Identity
+  (
+  )
 import Data.Inject
-import Data.Int hiding (fromString)
+  (
+  )
+import Data.Int
+  ( ceil
+  , even
+  , floor
+  , fromNumber
+  , fromString
+  , odd
+  , round
+  , toNumber
+  )
 import Data.Int.Bits
-import Data.Lazy hiding (Lazy (), defer)
+  ( complement
+  , shl
+  , shr
+  , zshr
+  , (.&.)
+  , (.^.)
+  , (.|.)
+  )
+import Data.Lazy
+  ( force
+  -- , Lazy ()
+  -- , defer
+  )
+import Data.List
+  ( List (Cons, Nil)
+  -- , alterAt
+  -- , catMaybes
+  -- , concat
+  -- , concatMap
+  -- , delete
+  -- , deleteAt
+  -- , deleteBy
+  -- , drop
+  -- , dropWhile
+  -- , elemIndex
+  -- , elemLastIndex
+  -- , filter
+  -- , filterM
+  -- , findIndex
+  -- , findLastIndex
+  -- , foldM
+  , fromList
+  -- , group
+  -- , group'
+  -- , groupBy
+  -- , head
+  -- , index
+  -- , init
+  -- , insert
+  -- , insertAt
+  -- , insertBy
+  -- , intersect
+  -- , intersectBy
+  -- , last
+  -- , length
+  -- , many
+  -- , mapMaybe
+  -- , modifyAt
+  -- , nub
+  -- , nubBy
+  -- , null
+  -- , range
+  -- , replicate
+  -- , replicateM
+  -- , reverse
+  -- , singleton
+  -- , slice
+  -- , snoc
+  -- , some
+  -- , sort
+  -- , sortBy
+  -- , span
+  -- , tail
+  -- , take
+  -- , takeWhile
+  , toList
+  -- , uncons
+  -- , union
+  -- , unionBy
+  -- , unzip
+  -- , updateAt
+  -- , zip
+  -- , zipWith
+  -- , zipWithA
+  -- , (:)
+  -- , (!!)
+  -- , (..)
+  -- , (\\)
+  )
+import Data.List.Lazy
+  ( -- List (..)
```

Comment:
I don't like how this ended up. I wish extra commas were optional in import lists so I could do something like this:

``` purescript
import Data.List.Lazy (
  -- , List (..)
  -- ...
)
```